backend/src/openarchiefbeheer/destruction/models.py

```diff
@@ -244,6 +253,24 @@ def generate_destruction_report(self) -> None:
 
         self.save()
 
+    def create_report_zaak(self) -> None:
```

Comment:
We discussed this but not sure: weren't the internal_results supposed to be cleared after the report zaak is created?